src/gc/gc.d

```diff
@@ -23,6 +23,8 @@ module gc.gc;
 //debug = LOGGING;              // log allocations / frees
 //debug = MEMSTOMP;             // stomp on memory
 //debug = SENTINEL;             // add underrun/overrrun protection
+                                // note: this needs to be enabled globally in the makefiles
+                                // (-debug=SENTINEL) instead of uncommented here.
```

Comment:
I think you should restrict the comment to building the unittests, e.g. ".. when building druntime unittests".